src/Approximations/template_directions.jl

```diff
@@ -330,3 +337,127 @@ end
 
 end # @eval
 end # if
+
+# ==================================================
+# Spherical directions
+# ==================================================
+
+@inline _isapprox_included(d, S) = any(isapproxzero.([norm(d - si, Inf) for si in S])) 
+
+"""
+    SphericalDirections <: AbstractDirections{Float64}
+
+Spherical directions representation.
+
+### Fields
+
+- `Nθ`    -- length of the partition of the acimutal angle
+- `Nφ`    -- length of the partition of the polar angle
+- `stack` -- list of computed directions
+
+### Notes
+
+The `SphericalDirections` constructor provides a sample of the unit sphere
+in ``\\mathbb{R}^3``, which is parameterized by the acimutal and polar angles
```

Comment:
> acimutal

In English this seems to be called [azimuthal](https://en.wikipedia.org/wiki/Azimuth) (occurs at least once more below).